woocommerce/woocommerce-ios
# Update lock icon

Closes: #11619


## Description

- changing lock icon to be system/symbols icon `lock.fill`

## Testing instructions

- open a completed order and try to edit it
- check that the correct icon is shown
- expand bottom sheet and check that the icon is shown in parts like Shipping, Coupons etc...

## Screenshots


| Before      | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-01-17 at 13 01 11](https://github.com/woocommerce/woocommerce-ios/assets/6242034/dd085f19-0312-426f-a536-e2fa2d710954)      | ![Simulator Screenshot - iPhone 15 Pro - 2024-01-17 at 12 57 59](https://github.com/woocommerce/woocommerce-ios/assets/6242034/4d01b337-4bb7-4361-a7b6-7e04d8f7bf91)       |

---
